velox/docs/functions/presto/math.rst

```diff
@@ -266,6 +266,10 @@ Probability Functions: cdf
     The mean and value must be real values and the standard deviation must be a real and
     positive value (all of type DOUBLE).
 
+.. function:: gamma_cdf(shape, scale, value) -> double
```

Comment:
Please move this function to the right alphabetical order in the list.

Reply:
Thanks @aditi-pandit . I have rearranged the function in the alphabetical order.